Error when using both KubeJS and The Big Bench Theory

OS: Void Linux
Java: 1.8.0_272

The following mods (& versions are used):
- Architectury 1.7.117
- The Big Bench Theory 1.0.1
- Fabric API 0.30.3+1.16
- KubeJS 1605.3.6-build.170
- Rhino 1.7.13-build.14

When loading all the mods listed, the game crashes during loading:

<details>
    <summary>Click to expand!</summary>

```
MultiMC version: 0.6.11-custom


Minecraft folder is:
/home/florian/.multimc/instances/test/.minecraft


Java path is:
/usr/lib/jvm/java-1.8-openjdk/jre/bin/java


Java is version 1.8.0_272, using 64-bit architecture.


Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz
Intel Corporation UHD Graphics 630 (Mobile)

DeviceName: Onboard - Video

Subsystem: CLEVO/KAPOK Computer Device 8555

OpenGL version string: 4.6 (Compatibility Profile) Mesa 20.3.4

Main Class:
  net.fabricmc.loader.launch.knot.KnotClient

Native path:
  /home/florian/.multimc/instances/test/natives

Traits:
traits XR:Initial
traits FirstThreadOnMacOS

Libraries:
  /home/florian/.multimc/libraries/org/lwjgl/lwjgl-glfw/3.2.2/lwjgl-glfw-3.2.2.jar
  /home/florian/.multimc/libraries/org/lwjgl/lwjgl-jemalloc/3.2.2/lwjgl-jemalloc-3.2.2.jar
  /home/florian/.multimc/libraries/org/lwjgl/lwjgl-openal/3.2.2/lwjgl-openal-3.2.2.jar
  /home/florian/.multimc/libraries/org/lwjgl/lwjgl-opengl/3.2.2/lwjgl-opengl-3.2.2.jar
  /home/florian/.multimc/libraries/org/lwjgl/lwjgl-stb/3.2.2/lwjgl-stb-3.2.2.jar
  /home/florian/.multimc/libraries/org/lwjgl/lwjgl-tinyfd/3.2.2/lwjgl-tinyfd-3.2.2.jar
  /home/florian/.multimc/libraries/org/lwjgl/lwjgl/3.2.2/lwjgl-3.2.2.jar
  /home/florian/.multimc/libraries/com/mojang/patchy/1.1/patchy-1.1.jar
  /home/florian/.multimc/libraries/oshi-project/oshi-core/1.1/oshi-core-1.1.jar
  /home/florian/.multimc/libraries/net/java/dev/jna/jna/4.4.0/jna-4.4.0.jar
  /home/florian/.multimc/libraries/net/java/dev/jna/platform/3.4.0/platform-3.4.0.jar
  /home/florian/.multimc/libraries/com/ibm/icu/icu4j/66.1/icu4j-66.1.jar
  /home/florian/.multimc/libraries/com/mojang/javabridge/1.0.22/javabridge-1.0.22.jar
  /home/florian/.multimc/libraries/net/sf/jopt-simple/jopt-simple/5.0.3/jopt-simple-5.0.3.jar
  /home/florian/.multimc/libraries/io/netty/netty-all/4.1.25.Final/netty-all-4.1.25.Final.jar
  /home/florian/.multimc/libraries/com/google/guava/guava/21.0/guava-21.0.jar
  /home/florian/.multimc/libraries/org/apache/commons/commons-lang3/3.5/commons-lang3-3.5.jar
  /home/florian/.multimc/libraries/commons-io/commons-io/2.5/commons-io-2.5.jar
  /home/florian/.multimc/libraries/commons-codec/commons-codec/1.10/commons-codec-1.10.jar
  /home/florian/.multimc/libraries/com/mojang/brigadier/1.0.17/brigadier-1.0.17.jar
  /home/florian/.multimc/libraries/com/mojang/datafixerupper/4.0.26/datafixerupper-4.0.26.jar
  /home/florian/.multimc/libraries/com/google/code/gson/gson/2.8.0/gson-2.8.0.jar
  /home/florian/.multimc/libraries/com/mojang/authlib/2.1.28/authlib-2.1.28.jar
  /home/florian/.multimc/libraries/org/apache/commons/commons-compress/1.8.1/commons-compress-1.8.1.jar
  /home/florian/.multimc/libraries/org/apache/httpcomponents/httpclient/4.3.3/httpclient-4.3.3.jar
  /home/florian/.multimc/libraries/commons-logging/commons-logging/1.1.3/commons-logging-1.1.3.jar
  /home/florian/.multimc/libraries/org/apache/httpcomponents/httpcore/4.3.2/httpcore-4.3.2.jar
  /home/florian/.multimc/libraries/it/unimi/dsi/fastutil/8.2.1/fastutil-8.2.1.jar
  /home/florian/.multimc/libraries/org/apache/logging/log4j/log4j-api/2.8.1/log4j-api-2.8.1.jar
  /home/florian/.multimc/libraries/org/apache/logging/log4j/log4j-core/2.8.1/log4j-core-2.8.1.jar
  /home/florian/.multimc/libraries/com/mojang/text2speech/1.11.3/text2speech-1.11.3.jar
  /home/florian/.multimc/libraries/net/fabricmc/intermediary/1.16.5/intermediary-1.16.5.jar
  /home/florian/.multimc/libraries/net/fabricmc/tiny-mappings-parser/0.2.2.14/tiny-mappings-parser-0.2.2.14.jar
  /home/florian/.multimc/libraries/net/fabricmc/sponge-mixin/0.8.2+build.24/sponge-mixin-0.8.2+build.24.jar
  /home/florian/.multimc/libraries/net/fabricmc/tiny-remapper/0.3.0.70/tiny-remapper-0.3.0.70.jar
  /home/florian/.multimc/libraries/net/fabricmc/access-widener/1.0.0/access-widener-1.0.0.jar
  /home/florian/.multimc/libraries/net/fabricmc/fabric-loader-sat4j/2.3.5.4/fabric-loader-sat4j-2.3.5.4.jar
  /home/florian/.multimc/libraries/com/google/jimfs/jimfs/1.2-fabric/jimfs-1.2-fabric.jar
  /home/florian/.multimc/libraries/org/ow2/asm/asm/9.0/asm-9.0.jar
  /home/florian/.multimc/libraries/org/ow2/asm/asm-analysis/9.0/asm-analysis-9.0.jar
  /home/florian/.multimc/libraries/org/ow2/asm/asm-commons/9.0/asm-commons-9.0.jar
  /home/florian/.multimc/libraries/org/ow2/asm/asm-tree/9.0/asm-tree-9.0.jar
  /home/florian/.multimc/libraries/org/ow2/asm/asm-util/9.0/asm-util-9.0.jar
  /home/florian/.multimc/libraries/net/fabricmc/fabric-loader/0.11.1/fabric-loader-0.11.1.jar
  /home/florian/.multimc/libraries/com/mojang/minecraft/1.16.5/minecraft-1.16.5-client.jar

Native libraries:
  /home/florian/.multimc/libraries/org/lwjgl/lwjgl-glfw/3.2.2/lwjgl-glfw-3.2.2-natives-linux.jar
  /home/florian/.multimc/libraries/org/lwjgl/lwjgl-jemalloc/3.2.2/lwjgl-jemalloc-3.2.2-natives-linux.jar
  /home/florian/.multimc/libraries/org/lwjgl/lwjgl-openal/3.2.2/lwjgl-openal-3.2.2-natives-linux.jar
  /home/florian/.multimc/libraries/org/lwjgl/lwjgl-opengl/3.2.2/lwjgl-opengl-3.2.2-natives-linux.jar
  /home/florian/.multimc/libraries/org/lwjgl/lwjgl-stb/3.2.2/lwjgl-stb-3.2.2-natives-linux.jar
  /home/florian/.multimc/libraries/org/lwjgl/lwjgl-tinyfd/3.2.2/lwjgl-tinyfd-3.2.2-natives-linux.jar
  /home/florian/.multimc/libraries/org/lwjgl/lwjgl/3.2.2/lwjgl-3.2.2-natives-linux.jar
  /home/florian/.multimc/libraries/com/mojang/text2speech/1.11.3/text2speech-1.11.3-natives-linux.jar

Mods:
  [✔️] architectury-1.7.117-fabric
  [✔️] big-bench-theory-1.0.1
  [✔️] fabric-api-0.30.3+1.16
  [✔️] kubejs-1605.3.6-build.170-fabric
  [✔️] rhino-1.7.13-build.14

Params:
  --username  --version MultiMC5 --gameDir /home/florian/.multimc/instances/test/.minecraft --assetsDir /home/florian/.multimc/assets --assetIndex 1.16 --uuid  --accessToken  --userType  --versionType release

Window size: 854 x 480

Java Arguments:
[-Xnoclassgc, -XX:+UnlockExperimentalVMOptions, -XX:+AggressiveOpts, -XX:MaxGCPauseMillis=50, -XX:+UseShenandoahGC, -XX:ConcGCThreads=3, -XX:+UseFastAccessorMethods, -XX:AllocatePrefetchStyle=2, -XX:+OptimizeStringConcat, -XX:+AlwaysPreTouch, -da, -dsa, -XX:CICompilerCount=3, -XX:CompileThreshold=20000, -Xms5000m, -Xmx5000m, -Duser.language=en]


Minecraft process ID: 23644


Using onesix launcher.

[21:38:22] [main/INFO]: Loading for game Minecraft 1.16.5
[21:38:23] [main/INFO]: [FabricLoader] Loading 50 mods: fabric-renderer-api-v1@0.4.1+ca58154a7d, fabric-keybindings-v0@0.2.1+ca58154a7d, rhino@1.7.13-build.14, kubejs@1605.3.6-build.170, fabricloader@0.11.1, fabric-structure-api-v1@1.1.4+ca58154a7d, fabric-containers-v0@0.1.11+9354966b7d, fabric-dimensions-v1@2.0.6+9354966b7d, fabric-object-builder-api-v1@1.9.4+9354966b7d, java@8, bigbenchtheory@1.0.1, fabric-game-rule-api-v1@1.0.6+ca58154a7d, fabric-api-base@0.2.1+9354966b7d, fabric-rendering-data-attachment-v1@0.1.5+ca58154a7d, fabric-textures-v0@1.0.6+ca58154a7d, fabric-rendering-fluids-v1@0.1.13+ca58154a7d, fabric-blockrenderlayer-v1@1.1.5+ca58154a7d, fabric-lifecycle-events-v1@1.2.1+ca58154a7d, fabric-renderer-registries-v1@2.2.1+ca58154a7d, fabric-loot-tables-v1@1.0.2+ca58154a7d, fabric@0.30.3+1.16, fabric-particles-v1@0.2.4+ca58154a7d, fabric-networking-v0@0.3.2+ca58154a7d, fabric-networking-blockentity-v0@0.2.8+ca58154a7d, minecraft@1.16.5, fabric-screen-handler-api-v1@1.1.5+9354966b7d, fabric-command-api-v1@1.0.10+ca58154a7d, fabric-screen-api-v1@1.0.0+c045166c7d, fabric-tool-attribute-api-v1@1.2.6+ca58154a7d, fabric-renderer-indigo@0.4.4+ca58154a7d, fabric-events-interaction-v0@0.4.2+ca58154a7d, fabric-crash-report-info-v1@0.1.3+ca58154a7d, fabric-item-api-v1@1.2.1+ca58154a7d, fabric-entity-events-v1@1.0.3+ca58154a7d, fabric-rendering-v0@1.1.2+ca58154a7d, fabric-key-binding-api-v1@1.0.4+9354966b7d, fabric-rendering-v1@1.5.1+ca58154a7d, fabric-resource-loader-v0@0.4.2+ca58154a7d, fabric-content-registries-v0@0.2.1+ca58154a7d, fabric-tag-extensions-v0@1.1.1+ca58154a7d, fabric-biome-api-v1@3.1.1+ca58154a7d, architectury@1.7.117, fabric-commands-v0@0.2.2+ca58154a7d, fabric-registry-sync-v0@0.7.4+ca58154a7d, fabric-mining-levels-v0@0.1.3+ca58154a7d, fabric-networking-api-v1@1.0.1+ca58154a7d, fabric-events-lifecycle-v0@0.2.1+ca58154a7d, fabric-item-groups-v0@0.2.3+ca58154a7d, fabric-models-v0@0.2.1+ca58154a7d, fabric-object-builders-v0@0.7.2+ca58154a7d
[21:38:23] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.2 Source=file:/home/florian/.multimc/libraries/net/fabricmc/sponge-mixin/0.8.2+build.24/sponge-mixin-0.8.2+build.24.jar Service=Knot/Fabric Env=CLIENT
[21:38:28] [main/INFO]: Environment: authHost='https://authserver.mojang.com', accountsHost='https://api.mojang.com', sessionHost='https://sessionserver.mojang.com', servicesHost='https://api.minecraftservices.com', name='PROD'
[21:38:29] [main/INFO]: Setting user: liketechnik2000
[21:38:29] [main/INFO]: Hello, World! (You will only see this line once in console, during startup)
[21:38:29] [main/INFO]: Loaded script startup_scripts:script.js in 0.097 s
[21:38:29] [main/INFO]: Loaded 1/1 KubeJS startup scripts in 0.409 s
[21:38:30] [main/INFO]: Hello, World! (You will see this line every time client resources reload)
[21:38:30] [main/INFO]: Loaded script client_scripts:script.js in 0.016 s
[21:38:30] [main/INFO]: Loaded 1/1 KubeJS client scripts in 0.039 s
[21:38:30] [main/INFO]: Initializing mod
[21:38:30] [main/INFO]: Initializing tags
[21:38:30] [main/INFO]: Initializing screen handlers
[21:38:30] [main/INFO]: Initializing blocks
---- Minecraft Crash Report ----
// I just don't know what went wrong :(

Time: 2/22/21 9:38 PM
Description: Initializing game

java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'kubejs'!
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke0(EntrypointUtils.java:53)
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke(EntrypointUtils.java:36)
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointClient.start(EntrypointClient.java:32)
	at net.minecraft.class_310.<init>(class_310.java:437)
	at net.minecraft.client.main.Main.main(Main.java:177)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:226)
	at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:139)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:27)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:196)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:231)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)
Caused by: java.lang.RuntimeException: java.lang.NoSuchFieldError: EVENT
	at dev.latvian.kubejs.fabric.KubeJSFabric.onInitialize(KubeJSFabric.java:38)
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke0(EntrypointUtils.java:50)
	... 19 more
Caused by: java.lang.NoSuchFieldError: EVENT
	at com.github.platymemo.bigbenchtheory.compat.kubejs.BigBenchKubeJS.onKubeJSInitialization(BigBenchKubeJS.java:24)
	at dev.latvian.kubejs.fabric.KubeJSFabric.lambda$onInitialize$0(KubeJSFabric.java:28)
	at java.util.ArrayList.forEach(ArrayList.java:1259)
	at dev.latvian.kubejs.fabric.KubeJSFabric.onInitialize(KubeJSFabric.java:27)
	... 20 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke0(EntrypointUtils.java:53)
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke(EntrypointUtils.java:36)
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointClient.start(EntrypointClient.java:32)
	at net.minecraft.class_310.<init>(class_310.java:437)

-- Initialization --
Details:
Stacktrace:
	at net.minecraft.client.main.Main.main(Main.java:177)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:226)
	at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:139)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:27)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:196)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:231)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)

-- System Details --
Details:
	Minecraft Version: 1.16.5
	Minecraft Version ID: 1.16.5
	Operating System: Linux (amd64) version 5.10.17_1
	Java Version: 1.8.0_272, Void
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Void
	Memory: 3249263240 bytes (3098 MB) / 5242880000 bytes (5000 MB) up to 5242880000 bytes (5000 MB)
	CPUs: 12
	JVM Flags: 14 total; -Xnoclassgc -XX:+UnlockExperimentalVMOptions -XX:+AggressiveOpts -XX:MaxGCPauseMillis=50 -XX:+UseShenandoahGC -XX:ConcGCThreads=3 -XX:+UseFastAccessorMethods -XX:AllocatePrefetchStyle=2 -XX:+OptimizeStringConcat -XX:+AlwaysPreTouch -XX:CICompilerCount=3 -XX:CompileThreshold=20000 -Xms5000m -Xmx5000m
	Fabric Mods: 
		architectury: Architectury 1.7.117
		bigbenchtheory: The Big Bench Theory 1.0.1
		fabric: Fabric API 0.30.3+1.16
		fabric-api-base: Fabric API Base 0.2.1+9354966b7d
		fabric-biome-api-v1: Fabric Biome API (v1) 3.1.1+ca58154a7d
		fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.5+ca58154a7d
		fabric-command-api-v1: Fabric Command API (v1) 1.0.10+ca58154a7d
		fabric-commands-v0: Fabric Commands (v0) 0.2.2+ca58154a7d
		fabric-containers-v0: Fabric Containers (v0) 0.1.11+9354966b7d
		fabric-content-registries-v0: Fabric Content Registries (v0) 0.2.1+ca58154a7d
		fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.1.3+ca58154a7d
		fabric-dimensions-v1: fabric-dimensions-v1 2.0.6+9354966b7d
		fabric-entity-events-v1: Fabric Entity Events (v1) 1.0.3+ca58154a7d
		fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.2+ca58154a7d
		fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.1+ca58154a7d
		fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.6+ca58154a7d
		fabric-item-api-v1: Fabric Item API (v1) 1.2.1+ca58154a7d
		fabric-item-groups-v0: Fabric Item Groups (v0) 0.2.3+ca58154a7d
		fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.4+9354966b7d
		fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.1+ca58154a7d
		fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 1.2.1+ca58154a7d
		fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.0.2+ca58154a7d
		fabric-mining-levels-v0: Fabric Mining Levels (v0) 0.1.3+ca58154a7d
		fabric-models-v0: Fabric Models (v0) 0.2.1+ca58154a7d
		fabric-networking-api-v1: Fabric Networking API (v1) 1.0.1+ca58154a7d
		fabric-networking-blockentity-v0: Fabric Networking Block Entity (v0) 0.2.8+ca58154a7d
		fabric-networking-v0: Fabric Networking (v0) 0.3.2+ca58154a7d
		fabric-object-builder-api-v1: Fabric Object Builder API (v1) 1.9.4+9354966b7d
		fabric-object-builders-v0: Fabric Object Builders (v0) 0.7.2+ca58154a7d
		fabric-particles-v1: Fabric Particles (v1) 0.2.4+ca58154a7d
		fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.7.4+ca58154a7d
		fabric-renderer-api-v1: Fabric Renderer API (v1) 0.4.1+ca58154a7d
		fabric-renderer-indigo: Fabric Renderer - Indigo 0.4.4+ca58154a7d
		fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 2.2.1+ca58154a7d
		fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.1.5+ca58154a7d
		fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 0.1.13+ca58154a7d
		fabric-rendering-v0: Fabric Rendering (v0) 1.1.2+ca58154a7d
		fabric-rendering-v1: Fabric Rendering (v1) 1.5.1+ca58154a7d
		fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.4.2+ca58154a7d
		fabric-screen-api-v1: Fabric Screen API (v1) 1.0.0+c045166c7d
		fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.1.5+9354966b7d
		fabric-structure-api-v1: Fabric Structure API (v1) 1.1.4+ca58154a7d
		fabric-tag-extensions-v0: Fabric Tag Extensions (v0) 1.1.1+ca58154a7d
		fabric-textures-v0: Fabric Textures (v0) 1.0.6+ca58154a7d
		fabric-tool-attribute-api-v1: Fabric Tool Attribute API (v1) 1.2.6+ca58154a7d
		fabricloader: Fabric Loader 0.11.1
		java: OpenJDK 64-Bit Server VM 8
		kubejs: KubeJS 1605.3.6-build.170
		minecraft: Minecraft 1.16.5
		rhino: Rhino 1.7.13-build.14
	Launched Version: MultiMC5
	Backend library: LWJGL version 3.2.2 build 10
	Backend API: NO CONTEXT
	GL Caps: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	CPU: <unknown>
#@!@# Game crashed! Crash report saved to: #@!@# /home/florian/.multimc/instances/test/.minecraft/crash-reports/crash-2021-02-22_21.38.30-client.txt
Process exited with code 255.
Clipboard copy at: 22 Feb 2021 21:47:32 +0100
```

</details>

Removing KubeJS fixes the crash.

The problem reoccurs if I add another mod which depends on KubeJS, which will not find the KubeJS bundled with The Big Bench Theory.

I'm already preparing a PR, but wanted to open this issue first for the sake of completeness.

--- a/build.gradle
+++ b/build.gradle
@@ -52,9 +52,8 @@ dependencies {
     modImplementation "de.siphalor:nbtcrafting-1.16:2+"
 
     // KubeJS Integration
-    // since this was like the only way to get it to work.
-    modImplementation "com.github.Technici4n:KubeJS-Fabric:${project.kubejs_commit}"
-    include "com.github.Technici4n:KubeJS-Fabric:${project.kubejs_commit}"
+    modImplementation "curse.maven:kubejs-${project.kubejs_project}:${project.kubejs_file}"
+	modImplementation "curse.maven:architectury-api-${project.architectury_project}:${project.architectury_file}"
 }
 
 processResources {

--- a/gradle.properties
+++ b/gradle.properties
@@ -17,4 +17,11 @@ org.gradle.jvmargs=-Xmx1G
 	fabric_version=0.29.3+1.16
 	modmenu_version=1.14.13+build.22
 	rei_version=5.8.10
-	kubejs_commit=849cc3c
+
+# Kube JS (loaded via https://www.cursemaven.com/)
+# project id from curseforge
+kubejs_project=395864
+# file id from curseforge
+kubejs_file=3211038
+architectury_project=419697
+architectury_file=3208685

--- a/src/main/java/com/github/platymemo/bigbenchtheory/compat/kubejs/BigBenchKubeJS.java
+++ b/src/main/java/com/github/platymemo/bigbenchtheory/compat/kubejs/BigBenchKubeJS.java
@@ -4,7 +4,6 @@ import com.google.gson.JsonArray;
 import com.google.gson.JsonElement;
 import com.google.gson.JsonObject;
 import dev.latvian.kubejs.KubeJSInitializer;
-import dev.latvian.kubejs.compat.AstromineRecipeEventHandler;
 import dev.latvian.kubejs.item.ItemStackJS;
 import dev.latvian.kubejs.item.ingredient.IngredientJS;
 import dev.latvian.kubejs.recipe.RecipeExceptionJS;

